edm4hep.yaml

```diff
@@ -781,6 +781,19 @@ datatypes:
     OneToOneRelations:
       - edm4hep::Track track           // the corresponding track
 
+  #-------------  RecTof
+  edm4hep::RecTof:
+    Description: "Reconstructed TOF (time of flight) info."
+    Author: "EDM4hep authors"
+    Members:
+      - float time [ns]                      // time measurement
+      - std::array<float, 5> timeExp [ns]    // expected time for e(0),mu(1),pi(2),K(3),p(4)
+      - float sigma                          // time resolution
+      - std::array<float, 5> pathLength [mm] // length of flight for e(0),mu(1),pi(2),K(3),p(4)
+      - edm4hep::Vector3d position [mm]      // extrapolated hit position
```

Comment:
I would potentially re-order these to keep the particle species bits and the "measured" bits together, e.g.
- `time`
- `sigma`
- `position`

`sigma` still needs a unit, which, I would assume is `ps`?

Is the extrapolated hit position necessary here? I assume you need it for the calculation of the track length, but do you still need it afterwards?

Reply:
I would also propose `sigma` -> `timeError`, which is similar to other places where we have value and valueError

Reply:
How about renaming `sigma` to something like `resolution`

Reply:
https://github.com/search?q=repo%3Akey4hep%2FEDM4hep+error+language%3AYAML&type=code&l=YAML